data/blog/funnel-101.mdx

```diff
@@ -0,0 +1,264 @@
+---
+title: 'Funnel 101: sharing your local developer preview with the world'
+date: 2023-03-10
+authors: ['xe']
+tags:
+  - funnel
+summary: 'Tailscale Funnel lets you share a local service with the internet. Today Xe covers how you can use this productively by sharing the development copy of an application to the world.'
```

Comment:
I think we might need an `images` key in here to populate the social images correctly. @Xe I presume you want the hero image as the share? Otherwise I can generate a generic text image